packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html

```diff
@@ -27,15 +27,17 @@
   </form>
 
   <amplify-slot name="sign-in-footer" [context]="context">
-    <button
-      amplify-button
-      fontWeight="normal"
-      size="small"
-      variation="link"
-      fullWidth="true"
-      (click)="authenticator.toResetPassword()"
-    >
-      {{ forgotPasswordText }}
-    </button>
+    <div data-amplify-footer>
```

Comment:
We use default styles under the slot, so that these opinionated slots are gone if customers override it.

One cool thing they can do is to put `data-amplify-footer`, and they'll get back the styles!